src/rules/no-useless-template-literals.ts

```diff
@@ -33,18 +33,16 @@ const rule: Rule.RuleModule = {
       pos: ESTree.SourceLocation,
       node: ESTree.TaggedTemplateExpression
     ): ESTree.Expression | null => {
-      for (let i = 0; i < node.quasi.quasis.length; i++) {
-        const quasi = node.quasi.quasis[i];
+      for (const expr of node.quasi.expressions) {
         if (
-          quasi.loc &&
-          ((pos.start.line > quasi.loc.start.line &&
-            pos.end.line < quasi.loc.end.line) ||
-            (pos.start.line === quasi.loc.start.line &&
-              pos.start.column >= quasi.loc.start.column) ||
-            (pos.start.line === quasi.loc.end.line &&
-              pos.start.column <= quasi.loc.end.column))
+          expr.loc &&
+          expr.loc.start.line === pos.start.line &&
+          expr.loc.start.column > pos.start.column &&
+          ((expr.loc.end.line === pos.end.line &&
+            expr.loc.end.column <= pos.end.column) ||
+            expr.loc.end.line < pos.end.line)
```

Comment:
This (`expr.loc.end.line < pos.end.line`) was the one branch of logic that wasn't getting hit by coverage.

Reply:
its very unlikely to happen i think, if ever.

```html
<div foo=${
v}
></div>
```

might cause it maybe, but i bet the parser just collapses the whitespace and it ends up on the last line

Reply:
Yeah, a test would be good 👍🏼